backends/sycl-ref/ceed-sycl-ref-basis.sycl.cpp

```diff
@@ -468,10 +460,18 @@ static int CeedBasisApplyNonTensor_Sycl(CeedBasis basis, const CeedInt num_elem,
   // Apply basis operation
   switch (eval_mode) {
     case CEED_EVAL_INTERP: {
-      CeedCallBackend(CeedBasisApplyNonTensorInterp_Sycl(data->sycl_queue, num_elem, is_transpose, impl, d_u, d_v));
+      if (is_transpose) {
+        CeedCallBackend(CeedBasisApplyInterp_Sycl<true>(data->sycl_queue, *impl->sycl_module, num_elem, impl, d_u, d_v));
+      } else {
+        CeedCallBackend(CeedBasisApplyInterp_Sycl<false>(data->sycl_queue, *impl->sycl_module, num_elem, impl, d_u, d_v));
```

Comment:
Doesn't like `true` or `false` either, different complaint though:
```
/home/jrwrigh/software/libCEED/backends/sycl-ref/ceed-sycl-ref-basis.sycl.cpp:464:82: error: no member named 'sycl_module' in 'CeedBasisNonTensor_Sycl'
  464 |         CeedCallBackend(CeedBasisApplyInterp_Sycl<true>(data->sycl_queue, *impl->sycl_module, num_elem, impl, d_u, d_v));
      |                                                                            ~~~~  ^
./include/ceed/backend.h:214:17: note: expanded from macro 'CeedCallBackend'
  214 |     int ierr_ = __VA_ARGS__;                                                     \
      |                 ^~~~~~~~~~~
```

Reply:
Oh, actually just looks like you're calling a different function now.

Reply:
That argument should have never actually been an int

Reply:
ugh

Reply:
Should be good now